adapters/insticator/insticator.go

```diff
@@ -0,0 +1,526 @@
+package insticator
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"math"
+	"net/http"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+// type adapter struct {
+// 	endpoint string
+// }
+
+type Ext struct {
+	Insticator impInsticatorExt `json:"insticator"`
+}
+
+type insticatorUserExt struct {
+	Eids    []openrtb2.EID  `json:"eids,omitempty"`
+	Data    json.RawMessage `json:"data,omitempty"`
+	Consent string          `json:"consent,omitempty"`
+}
+
+type impInsticatorExt struct {
+	AdUnitId    string `json:"adUnitId,omitempty"`
+	PublisherId string `json:"publisherId,omitempty"`
+}
+
+type adapter struct {
+	endpoint string
+}
+
+type reqExt struct {
+	Insticator *reqInsticatorExt `json:"insticator,omitempty"`
+}
+
+type reqInsticatorExt struct {
+	Caller []InsticatorCaller `json:"caller,omitempty"`
+}
+
+type InsticatorCaller struct {
+	Name    string `json:"name,omitempty"`
+	Version string `json:"version,omitempty"`
+}
+
+// CALLER Info used to track Prebid Server
+// as one of the hops in the request to exchange
+var CALLER = InsticatorCaller{"Prebid-Server", "n/a"}
```

Comment:
Accessibility scope of `CALLER` is public i.e can be accessed outside of insticator package. Should make it private as done below:

```
// caller Info used to track Prebid Server
// as one of the hops in the request to exchange
var caller = InsticatorCaller{"Prebid-Server", "n/a"}
```